index.html

```diff
@@ -76,14 +78,27 @@ <h2 class="text-white-50 mx-auto mt-2 mb-5">Machine Learning for
     <div class="container">
         <div class="row">
             <div class="col-lg-8 mx-auto">
-                <h2 class="text-white mb-4">Supporting the Cement Supply Chain
-                    with AI</h2>
-                <div class="text-white-50">We use Machine Learning and Bayesian
-                    Optimization to cut costs
-                    and reduce the CO2 footprint of the Cement Supply
-                    Chain.
+                <h2 class="text-white mb-4">Introduction</h2>
+                <div class="text-white-50">There are several decisions you have to take in the last step of cement
+                    production. You are trying to get the 2 day or 28 day strength right, to the target corridor agreed with your
+                    customer, while at the same time having to deal with fluctuating clinker properties and inconsistent
+                    slag and fly ash qualities. All of these also have different availabilities and prices. Maybe you
+                    even purchase these from different manufacturers, not knowing what exactly to expect. Now you need
+                    to decide recipes and mill targets, while also trying to produce cost optimally. We know how
+                    difficult that is.
+                    <br>
+                    <br>
+                    We have built algorithms and machine learning models from all available laboratory data
+                    to help you with these dicisions --
+                    combining your strengths (understanding the reason behind abnormalities) and the strengths of
```

Comment:
too many round brackets

Reply:
Please, give me the brackets!!! (they make sense here)